projects/schizophrenia/src/ExomePage/RegionViewer.js

```diff
@@ -175,11 +157,9 @@ export default connect(
     visibleVariants: finalFilteredVariants(state),
     screenSize: screenSize(state),
     variantFilter: variantFilter(state),
-    focusedVariant: focusedVariant(state),
```

Comment:
Removing this line makes the click handler on variant IDs in the table 10x faster (from ~300ms to ~30ms). With this line, changing the focused variant causes this SchizophreniaGeneViewer component to render, and its render method maps/filters over the variants list 5 times.